src/components/blockstackProfile.js

```diff
@@ -7,8 +7,9 @@ import {
 import { Avatar } from '@material-ui/core'
 import AccountIcon from '@material-ui/icons/AccountBox'
 import { BlockstackButton } from 'react-blockstack-button'
-import { navigateTo } from 'gatsby'
 import { styles } from './layout'
+import { navigateTo } from 'gatsby'
```

Comment:
navigate should be used instead of naviageTo

Reply:
okay sure